.github/workflows/release.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.16.3
+        go-version: 1.19.3
```

Comment:
Based on (limited) CI observations, it appears that 1.18 stuff either always works or is less likely to fail. If true, consider running with 1.18 for now

Reply:
Ok

Reply:
Thus, if we restart CI enough times - we will get all green